### DISCUSSION TITLE
雲端賦能 Google Cloud

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#621] [General]
太神奇啦，既可以放大ai優勢，同時又在做環保，一兼二得呢!
https://www.cw.com.tw/article/5124226

**Comment by magicmicheal** [comment]
2023關鍵字：雲端賦能 Google Cloud 放大 AI 優勢，帶領企業轉型也助全球淨零

**Comment by awehao** [comment]
又是讚，又是眼睛搞得我好亂啊，想必同學和老師肯定都對此感到頭痛吧。隨著工業和科技的發展，人類對地球的危害也越來越大，這時我們就要省思，如何將美麗的地球恢復，減碳的出發點是好的，不過就像垃圾分類那樣，真正要改變的是我們，能否落實才是最重要的

**Comment by awkward-willy** [comment]
這次的 Covid - 19 疫情讓人們對科技產品的依賴程度提升，也因此造成許多服務蓬勃發展，而這些往往會需要各式各樣的雲端平台，也因此這些平台的成長與興盛是必然的結果，我們也應當多學習這些平台的使用，以順應未來潮流的趨勢。
